bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/installcomponents/messages.properties

```diff
@@ -0,0 +1,3 @@
+InstallComponents_InstallButton=Install
+InstallComponents_InstallButtonAlreadyAdded=Already Added
+InstallComponents_OpenReadmeButton=Open README
```

Comment:
Maybe we can change it to the user-friendly name `Details...` or `More info..`

Reply:
Done